### DISCUSSION TITLE
probleme with fast switching app using Alt+Tab (fedora 32)

### DISCUSSION THREAD

**Discussion by Perry-chouteau** [raysan5/raylib#2519] [Q&A] [answered]
I'm not sure if I can handle that but the window opened don't follow switching app priority using Alt+Tab on fedora.
The window is at first position when I'm watching it and switch to the last position when I'm watching an other app.

**Comment by raysan5** [comment, marked as answer]
@Perry-chouteau I don't know if this can be controlled from raylib, maybe from GLFW... In any case I don't know the origin of this issue.

**Comment by Nicholaus04** [comment]
This is different, you have never emailed me
